task3-make/Makefile

```diff
@@ -0,0 +1,78 @@
+CC =
+
+CHECKER = ../scripts/checkpatch.pl
+CHECKER_FLAGS = --no-tree -f
+
+PROGRAM = main
+
+EXTRA_LDFLAGS =
+EXTRA_CFLAGS  = -fdata-sections -ffunction-sections
+CPPFLAGS      =
+CFLAGS        =
+
+SRCDIR  = ../task1-simple-program
+SOURCES = $(SRCDIR)/main.c $(SRCDIR)/get_random_number.c
+HEADERS = $(SRCDIR)/get_random_number.h
+OBJS    = $(addsuffix .o, $(basename $(SOURCES)))
+
+DEBUG =
+
+ifeq ($(DEBUG),1)
```

Comment:
It is a common approach to have some specific task for make (e.g., "build" or whatever you like) with parameters.